spec/datadog/appsec/configuration/settings_spec.rb

```diff
@@ -551,7 +533,7 @@ def patcher
         end
 
         context 'when given a supported value' do
-          let(:track_user_events_mode) { :extended }
+          let(:track_user_events_mode) { 'extended' }
```

Comment:
There's a small change, where somewhere we are converting a `:extended` to a string, but now we just pass the string.
Is this material to the application running appsec, or simply a testing artifact?

Reply:
On the first iteration, we accepted either string or symbol as a possible value for `appsec.track_user_events.mode`. Internally we stored it as a string, and what ever check we did against those values expected a string.

https://github.com/DataDog/dd-trace-rb/blob/7a8771098170280b81c5e4acb9b6cab2393fbadd/lib/datadog/appsec/contrib/devise/event.rb#L41-L44

With this change, we signal that we only support string values. 

As a note, this feature hasn't ben release yet, so there are no users using it, so there is no breaking change or anything that could potentially affect users of appsec